reference-architecture/azure-ansible/bastion.sh

```diff
@@ -0,0 +1,337 @@
+
+# default storage plugin dependencies to install, by default the ceph and
+# glusterfs plugin dependencies will be installed, if available.
+osn_storage_plugin_deps=['iscsi']
+
+[masters]
+master1 openshift_node_labels="{'role': 'master'}"
+master2 openshift_node_labels="{'role': 'master'}"
+master3 openshift_node_labels="{'role': 'master'}"
+
+[etcd]
+master1
+master2
+master3
+
+[nodes]
+master1 openshift_node_labels="{'region':'master','zone':'default'}" 
+master2 openshift_node_labels="{'region':'master','zone':'default'}" 
+master3 openshift_node_labels="{'region':'master','zone':'default'}" 
+node[01:${NODECOUNT}] openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
+infranode1 openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+infranode2 openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+infranode3 openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
+
+[quotanodes]
+master1 openshift_node_labels="{'region':'master','zone':'default'}" 
+master2 openshift_node_labels="{'region':'master','zone':'default'}" 
+master3 openshift_node_labels="{'region':'master','zone':'default'}" 
+node[01:${NODECOUNT}] openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
+
+[misc]
+store1
+EOF
+
+
+cat <<EOF > /home/${AUSERNAME}/subscribe.yml
+---
+- hosts: all
+  vars:
+    description: "Subscribe OSE"
+  tasks:
+  - name: wait for .updateok
+    wait_for: path=/root/.updateok
+  - name: check connection
+    ping:
+  - name: Get rid of rhui repos
+    file: path=/etc/yum.repos.d/rh-cloud.repo state=absent
+  - name: Get rid of rhui Load balancers
+    file: path=/etc/yum.repos.d/rhui-load-balancers state=absent
+  - name: remove the RHUI package
+    yum: name=RHEL7 state=absent
+  - name: Get rid of old subs
+    shell: subscription-manager unregister
+    ignore_errors: yes
+  - name: register hosts
+    shell: subscription-manager register --username ${RHNUSERNAME} --password ${RHNPASSWORD} 
+    register: task_result
+    until: task_result.rc == 0
+    retries: 10
+    delay: 30
+    ignore_errors: yes
+  - name: attach sub
+    shell: subscription-manager attach --pool=$RHNPOOLID
+    register: task_result
+    until: task_result.rc == 0
+    retries: 10
+    delay: 30
+    ignore_errors: yes
+  - name: disable all repos
+    shell: subscription-manager repos --disable="*" 
+  - name: enable rhel7 repo
+    shell: subscription-manager repos --enable="rhel-7-server-rpms"
+  - name: enable extras repos
+    shell: subscription-manager repos --enable="rhel-7-server-extras-rpms"
+  - name: enable ose repos
+    shell: subscription-manager repos --enable="rhel-7-server-ose-3.3-rpms"
+  - name: install the latest version of PyYAML
+    yum: name=PyYAML state=latest
+  - name: Install the ose client
+    yum: name=atomic-openshift-clients state=latest
+  - name: Update all hosts
+    command: yum -y update
+    async: 1200
+    poll: 10
+  - name: Wait for Things to Settle
+    pause:  minutes=5
```

Comment:
It is installed in all the hosts except bastion.
So is there a requirement on the bastion host?